autoload/coverage/python.vim

```diff
@@ -34,21 +36,22 @@ function! coverage#python#GetCoveragePyProvider() abort
   endfunction
 
   function l:provider.GetCoverage(filename) abort
-    " Check coverage is importable and show a clear error otherwise.
-    try
-      python import coverage
-    catch /Vim(python):/
-      throw maktaba#error#NotFound(
-          \ "Couldn't import python coverage module. " .
-          \ 'Install coverage and try again.')
-    endtry
+    if !s:imported_python
+      try
+        call maktaba#python#ImportModule(s:plugin, 'vim_coverage')
+      catch /ERROR.*/
+          throw maktaba#error#NotFound(
+              \ "Couldn't import Python coverage module (".v:exception.'). ' .
```

Comment:
You can use %s here and pass v:exception as `[args...]` instead of concatenating.

Reply:
Thanks, pushed a fixup.